build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -267,9 +267,9 @@ public static BuildTargetResult GenerateMSbuildPropsFile(BuildTargetContext c)
             msbuildProps.AppendLine($"    <HostVersion>{hostVersion.LatestHostPolicyVersion.WithoutSuffix}</HostVersion>");
             msbuildProps.AppendLine($"    <HostResolverVersion>{hostVersion.LatestHostFxrVersion.WithoutSuffix}</HostResolverVersion>");
             msbuildProps.AppendLine($"    <HostPolicyVersion>{hostVersion.LatestHostVersion.WithoutSuffix}</HostPolicyVersion>");
```

Comment:
Yes, this should be swapped.

Should be:

`HostVersion>{hostVersion.LatestHostVersion.WithoutSuffix}</HostVersion>"`